### PR TITLE
Build against akka 2.5

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: docs/src/main/paradox/scala/http/compatibility-guidelines.md [hunk 1]
```diff
@@ -71,13 +68,13 @@ Maven
     <dependency>
       <groupId>com.typesafe.akka</groupId>
       <artifactId>akka-http_$scala.binary_version$</artifactId>
-      <version>$akka25.version$</version>
+      <version>$project.version$</version>
```
Comment (jrudolph) [inline docs/src/main/paradox/scala/http/compatibility-guidelines.md]: Oops, good catch. Did I recently introduce that?

Comment (raboof) [reply docs/src/main/paradox/scala/http/compatibility-guidelines.md]: Looks like it (62ed17d15), might be a merge thing?

**Comment by raboof** [comment]
PLS BUILD

**Comment by jrudolph** [comment]
@raboof what do you think about the docs suggestion?

**Comment by raboof** [comment]
Ah, good ones, I'll pick up those comments 👍

**Pull request merged**